tests/Jellyfin.Server.Implementations.Tests/IO/ManagedFileSystemTests.cs

```diff
@@ -24,22 +24,39 @@ public ManagedFileSystemTests()
         [InlineData("/Volumes/Library/Sample/Music/Playlists/", "../Beethoven/Misc/Moonlight Sonata.mp3", "/Volumes/Library/Sample/Music/Beethoven/Misc/Moonlight Sonata.mp3")]
         [InlineData("/Volumes/Library/Sample/Music/Playlists/", "../../Beethoven/Misc/Moonlight Sonata.mp3", "/Volumes/Library/Sample/Beethoven/Misc/Moonlight Sonata.mp3")]
         [InlineData("/Volumes/Library/Sample/Music/Playlists/", "Beethoven/Misc/Moonlight Sonata.mp3", "/Volumes/Library/Sample/Music/Playlists/Beethoven/Misc/Moonlight Sonata.mp3")]
-        public void MakeAbsolutePathCorrectlyHandlesRelativeFilePaths(
+        [InlineData("/Volumes/Library/Sample/Music/Playlists/", "/mnt/Beethoven/Misc/Moonlight Sonata.mp3", "/mnt/Beethoven/Misc/Moonlight Sonata.mp3")]
+        public void MakeAbsolutePathCorrectlyHandlesRelativeFilePathsOnUnixLike(
             string folderPath,
             string filePath,
             string expectedAbsolutePath)
         {
-            var generatedPath = _sut.MakeAbsolutePath(folderPath, filePath);
-
             if (OperatingSystem.IsWindows())
             {
-                var expectedWindowsPath = expectedAbsolutePath.Replace('/', '\\');
-                Assert.Equal(expectedWindowsPath, generatedPath.Split(':')[1]);
+                return;
             }
-            else
+
+            var generatedPath = _sut.MakeAbsolutePath(folderPath, filePath);
+            Assert.Equal(expectedAbsolutePath, generatedPath);
+        }
+
+        [Theory]
```

Comment:
Use `SkippableTheoryAttribute`

See `GetFileInfo_DanglingSymlink_ExistsFalse` test below on how it's used